build.ts

```diff
@@ -16,7 +16,7 @@ async function main() {
       format: "iife",
       globalName: "BlockBundle",
       minify: true,
-      external: ["fs", "path", "assert", "react", "react-dom"],
+      external: ["fs", "path", "assert", "react", "react-dom", "@primer/react"],
```

Comment:
For posterity, @jaked recommended this change to ensure that the bundled Block code was using the correct instance of `@primer/react` at runtime.

Reply:
reminder that you should make this change in `blocks-template` too

Reply:
reminder that you should make this change in `blocks-template` too

Reply:
@jaked Roger that! I'll do a follow-up PR on Monday :)